### PR TITLE
Create container.md

### LINKED ISSUE #2599: [ja] Localize Containers
Localize https://glossary.cncf.io/container/

### PR DESCRIPTION
**Pull request by ARYANis0001**
### Describe your changes
Localize Containers in Japanese

### Related issue number or link (ex: `resolves #issue-number`)
resolves #2599 

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/ja/container.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+title: コンテナ
+status: Completed
+category: テクノロジー
+tags: ["アプリケーション"、"基本"、""]
```
Comment (jihoon-seo) [inline content/ja/container.md:5]: Netlify build failed.
> error building site: assemble: "/opt/build/repo/content/ja/container.md:3:1": failed to unmarshal YAML: yaml: line 3: did not find expected "," or "]"

```suggestion
tags: ["アプリケーション","基本",""]
```

**Comment by ARYANis0001** [comment]
Yes I will update it

**Review by ARYANis0001 — COMMENTED** [review]
Updated

File: content/ja/container.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+## それは何ですか
+
+コンテナは、コンピュータのオペレーティング システムによって管理されるリソースと機能の制約を持つ実行中のプロセスです。
+コンテナー プロセスで使用できるファイルは、コンテナー イメージとしてパッケージ化されます。
+コンテナは同じマシン上で互いに隣接して実行されます。
+ただし、通常、オペレーティング システムは、個別のコンテナ プロセスが相互に対話することを妨げます。
+
+## 対処する問題
+
+コンテナーが利用可能になる前は、アプリケーションを実行するには別のマシンが必要でした。
```
Comment (naonishijima) [inline content/ja/container.md:17]: 日本語用語集では、コンテナーは長音をつけないことにしています。

File: content/ja/container.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+各マシンには独自のオペレーティング システムが必要であり、CPU、メモリ、ディスク容量が必要になります。
```
Comment (naonishijima) [inline content/ja/container.md:18]: ここの翻訳は難しいですね。ownを「独自」と訳すと独自OSと読めてしまうので、「自身」や「自らの」がよいと思いました。
「CPU、メモリ、ディスク容量が必要になります。」と一旦区切るよりも「CPU、メモリ、ディスク容量が、個々の〜」とした方が繋がりが良いと思いました。

File: content/ja/container.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+すべては個々のアプリケーションが機能するために必要です。
+さらに、オペレーティング システムのメンテナンス、アップグレード、起動もまた、大きな労力の源です。
```
Comment (naonishijima) [inline content/ja/container.md:20]: ここは好みの問題だとおもいますが、「大きな労力の源」よりも「労力の大きな原因」の方が自然だと思いました。

File: content/ja/container.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+## どのように役立つか
+
+コンテナは同じオペレーティング システムとそのマシン リソースを共有します。
+オペレーティング システムのリソース オーバーヘッドを分散し、物理マシンを効率的に使用します。
+この機能が可能になるのは、通常、コンテナー同士の相互作用が制限されているためです。
+これにより、より多くのアプリケーションを同じ物理マシン上で実行できるようになります。
+
+ただし、制限もあります。
+コンテナーは同じオペレーティング システムを共有するため、プロセスは代替プロセスよりも安全性が低いと考えられます。
```
Comment (naonishijima) [inline content/ja/container.md:30]: ここの長音も不要です。

**Review by naonishijima — COMMENTED** [review]
Thank you for your interest. However, I felt that you needed to become more familiar with the Japanese language. If you are still interested, I would appreciate it if you could deal with the comments I made.

**Comment by seokho-son** [comment]
Hi @ARYANis0001,

Thank you for your interest and contributions to the Glossary project.

As one of the maintainers of this repository, I want to address some concerns that have arisen during the review of your PRs. Reviewing localization PRs demands significant effort, and it's crucial that submissions meet high standards of quality. 

We've noticed that the content in some of your PRs doesn't seem to meet these standards, particularly in terms of language fluency and adherence to guidelines. Direct machine translations or low-effort submissions cannot be accepted, as they don't align with our commitment to quality.

Your contributions across various languages (Hindi, German, Japanese, Portuguese) as seen in https://github.com/cncf/glossary/pulls/ARYANis0001 have raised questions about language fluency. This observation is based on comments and feedback from several reviewers. Additionally, many of these PRs appear to deviate from our guidelines, showing inconsistencies and technical issues.

We value your participation, but we must also consider the workload of our reviewers. If you have any thoughts or responses to these concerns, please share them. Otherwise, we may need to close your open PRs to streamline the review process.

If you wish to resubmit, we encourage you to put forth your best effort, ensuring full compliance with our contribution guidelines and style. For localization contributions, please also refer to specific guidance from the relevant language team, as exemplified in https://github.com/cncf/glossary/pull/2407#issuecomment-1723456444.

**Comment by ARYANis0001** [comment]
Hey @seokho-son  
Hey @kumarankit999 
I want to express my sincere apologies for any inconvenience caused to the team members of CNCF/Glossary due to my raised PRs. I fully understand my mistake and acknowledge that I need to be more careful in the future. Going forward, I will ensure to thoroughly read the contributing guidelines and submit PRs in my native language, which I am fluent in and proficient in.

**Comment by seokho-son** [comment]
> Hey @seokho-son Hey @kumarankit999 I want to express my sincere apologies for any inconvenience caused to the team members of CNCF/Glossary due to my raised PRs. I fully understand my mistake and acknowledge that I need to be more careful in the future. Going forward, I will ensure to thoroughly read the contributing guidelines and submit PRs in my native language, which I am fluent in and proficient in.

Hi @ARYANis0001 
Thank you for your understanding and willingness to improve. We look forward to your continued contributions. :)

**Pull request closed without merging**